src/components/CreateArticlesForm.jsx

```diff
@@ -1,65 +1,77 @@
 import React, { useState } from "react";
-import { Form, Container } from "semantic-ui-react";
+import { Form, Container, Message } from "semantic-ui-react";
 import Article from "../modules/articles";
 
 const CreateArticlesForm = () => {
   const [message, setMessage] = useState("");
   const [selectedCategory, setSelectedCategory] = useState("");
+  const [image, setImage] = useState();
+
+  const toBase64 = (file) =>
+    new Promise((resolve, reject) => {
+      const reader = new FileReader();
+      reader.readAsDataURL(file);
+      reader.onload = () => resolve(reader.result);
+      reader.onerror = (error) => reject(error);
+    });
+
+  const selectImage = (e) => {
+    setImage(e.target.files[0]);
+  };
 
   const handleCategoryChange = (value) => {
-    setSelectedCategory(value);
+   setSelectedCategory(value);
+   return selectedCategory
   };
 
   const onSubmit = async (e) => {
     e.preventDefault();
-    const result = await Article.create(
-      e.target.title.value,
-      e.target.lead.value,
-      e.target.content.value,
-      selectedCategory
-    );
-    setMessage(result);
+    let { title, lead, content, encodedImage } = e.target;
+    if (image) {
+      encodedImage = await toBase64(image);
+    }
+    const response = await Article.create(title, lead, content, encodedImage);
+    setMessage(response);
   };
 
   return (
     <Container>
       <Form data-cy="create-article" id="create-article" onSubmit={onSubmit}>
         <Form.Group widths="equal" data-cy="form-article">
-          <Form.Input
-            fluid
-            label="Title"
-            placeholder="Title"
-            id="title"
-            data-cy="title"
-          />
-          <Form.Input
-            fluid
-            label="Lead"
-            placeholder="Lead"
-            data-cy="lead"
-            id="lead"
-          />
+          <Form.Input fluid label="Title" placeholder="Title" data-cy="title" />
+          <Form.Input fluid label="Lead" placeholder="Lead" data-cy="lead" />
```

Comment:
Consistency is important, in a file below (in this. PR) you use a new line for each attribute of a tag. Here, you keep everything inline. Please choose one and stick to it.

Reply:
this is made by the formatter, we have less inputs here so thats why the formatter makes it just oneline